entity.go

```diff
@@ -213,7 +213,7 @@ func (e *entity) MessageTypes() EntityMessageTypes {
 }
 
 func (e *entity) TypeName() string {
-	return typename.Of(e.ReflectType())
+	return goreflect.Of(e.ReflectType())
```

Comment:
```suggestion
	return goreflect.NameOf(e.ReflectType())
```

We should probably rename these functions now.